ts/screens/onboarding/EmailValidateScreen.tsx

```diff
@@ -65,19 +66,17 @@ export class EmailValidateScreen extends React.PureComponent<Props> {
     return (
       <TopScreenComponent
         goBack={this.handleGoBack}
-        headerTitle={I18n.t("onboarding.email.validation.headerTitle")}
-        title={I18n.t("onboarding.email.validation.title")}
+        headerTitle={I18n.t("email.validate.header")}
+        title={I18n.t("email.validate.title")}
         contextualHelp={{
-          title: I18n.t("onboarding.email.validation.title"),
-          body: () => (
-            <Markdown>{I18n.t("onboarding.email.validation.help")}</Markdown>
-          )
+          title: I18n.t("email.validate.title"),
+          body: () => <Markdown>{I18n.t("email.validate.help")}</Markdown>
```

Comment:
Try to move out this lamba to a const function

Reply:
Here is valid the same proposal introduced in the previous comment.

Reply:
Yes. I address the proposal as valid. You can change also this part 👍